# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/python-pyopenssl/no-download.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pyopenssl-23.3.0/setup.py.vanilla	2024-01-07 17:45:20.153568418 +0100
+++ pyopenssl-23.3.0/setup.py	2024-01-07 17:45:35.375096776 +0100
@@ -93,7 +93,6 @@
         packages=find_packages(where="src"),
         package_dir={"": "src"},
         install_requires=[
-            "cryptography>=41.0.5,<42",
         ],
         extras_require={
             "test": ["flaky", "pretend", "pytest>=3.0.1"],
